parsers/ENTSOE.py

```diff
@@ -104,9 +109,12 @@ class WindAndSolarProductionForecastTypes(Enum):
         "oil": ["B06"],
         "solar": ["B16"],
         "wind": ["B18", "B19"],
-        "unknown": ["B20", "B13", "B15"],
+        "unknown": ["B20", "B13", "B15", "B21", "B22", "B23", "B24"],
```

Comment:
This is a risky change, we currently don't know if AC link and DC link would duplicate the data we get from the exchange data.

We also don't know what Substation or Transformer mean in this context, if it just includes their consumption then unknown could become negative. Which would not be correct.

Reply:
That's fair - I thought of these too, but hoped that either someone knows better than me, or that this change could be verified in a test deployment first (assuming there is any actual data coming in with these types from ENTSO-E).

Reply:
The best way would be to test this locally, using the `poetry test_parser <zone>` command.

But if the data is not added to the zone in question then it becomes impossible to test and if it suddenly changes later a bunch of weird things can happen.